src/components/Admins.vue

```diff
@@ -24,7 +24,7 @@
 		<span :class="{ hidden: !isLoading }" class="icon icon-loading" />
 		<div v-for="admin in admins" :key="admin.id" class="admin-avatar-container">
 			<NcAvatar :user="admin.internal ? admin.id : null"
-				:display-name="admin.displayname"
```

Comment:
Oo This breaks the fallback of avatars?
https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAvatar/NcAvatar.vue#L281-L290

Reply:
Use the plus in the UI to see:

master | this PR
---|---
![Bildschirmfoto vom 2023-01-23 15-45-09](https://user-images.githubusercontent.com/213943/214068654-b6ec47a7-b471-41e4-8430-7f22951f6e69.png) | ![Bildschirmfoto vom 2023-01-23 15-45-25](https://user-images.githubusercontent.com/213943/214068660-01b12a38-0f42-4863-b82b-b53e0aeb290a.png)

Reply:
corrected, thanks!